testing/infra/terraform/modules/standalone_apm_server/outputs.tf

```diff
@@ -5,6 +5,11 @@ output "apm_secret_token" {
 }
 
 output "apm_server_url" {
-  value       = "${aws_instance.apm.public_ip}:${local.apm_port}"
+  value       = "http://${aws_instance.apm.public_ip}:${local.apm_port}"
```

Comment:
I wonder if it's important at all to test with HTTPS here.

Reply:
It's debatable but I think https shouldn't really matter here, since it shouldn't qualitatively change the resulting profile.